nginx/config.go

```diff
@@ -293,6 +293,17 @@ http {
 		proxy_pass {{$builderConfig.ServiceIP}}:2222;
 	}
 }{{ end }}
+
+{{ if $routerConfig.StatusEndpoint }}
+  sevrer {
```

Comment:
Hm, I think it would be easy to say that the router can assume a domain for itself, that could be the default vhost, if the `statusEndpoint` option is `true` there we could expose it under the default `/nginx_status`. Is that a viable approach? 

I imagine it would make some things easier, like explaining how to hit the router from outside. But now this is becoming a new thing => Adding a router domain, or something along those lines, && enabling status endpoint on default vhost if I'm not mistaken.